### ISSUE TITLE
Special Characters in Subtitles

### ISSUE THREAD

**Issue by fcrdossantos** [FaisalUmair/udemy-downloader-gui#178]
There's a bug with the special characters in subtitles like for example in Portuguese when they use the word "você" (you) and the legend is "voc" or "coração" (heart) and turns into "corao"

**Comment by FaisalUmair** [comment]
Yes, actually we are using a filename sanitation plugin before writing any file on disk. So this sort of issue is expected.

**Comment by fcrdossantos** [comment]
And there is anything you can do to solve this problem?
